src/pages/blog/starlingx-deploying-your-app-1.md

```diff
@@ -0,0 +1,309 @@
+---
+templateKey: blog-post
+title: Deploying your application to StarlingX
+author: Bruno Muniz
+date: 2023-10-13T12:00:00.042Z
+category:
+  - label: Features & Updates
+---
+
+Deploying your own application to StarlingX is as easy as any other Kubernetes
+deployment out there. In this blog post we'll show you the process with a
+simple demonstration application.<!- more ->
```

Comment:
```suggestion
simple demo application.<!- more ->
```

Reply:
It is very common to just say "demo application" or "demo app", using the shorter version also helps making the article easier to read.